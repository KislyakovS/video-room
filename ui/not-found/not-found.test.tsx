import { render } from '@testing-library/react';

import NotFound from './not-found';

describe('<NotFound>', () => {
    it('snapshot', () => {
        const { container } = render(<NotFound />);

        expect(container).toMatchSnapshot();
    });
})