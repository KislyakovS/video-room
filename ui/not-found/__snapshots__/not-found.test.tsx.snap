// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound> snapshot 1`] = `
<div>
  <section
    class="wrapper"
  >
    <span
      class="code"
    >
      404
    </span>
    <h3
      class="title"
    >
      Not Found
    </h3>
    <span>
      Sorry, we were unable to find that page.
    </span>
  </section>
</div>
`;
